ir/constant.h

```diff
@@ -33,6 +34,18 @@ class IntConst final : public Constant {
   auto getInt() const { return std::get_if<int64_t>(&val); }
 };
 
+class FloatConst final : public Constant {
+  std::variant<std::bitset<8>, std::bitset<16>, std::bitset<32>,
+               std::bitset<64>, std::bitset<128>, std::bitset<80>,
```

Comment:
ok, I was wondering why you used a bitset instead of a uint32_t and uint64_t, but this makes sense if you're going to need to represent 80 and 128 bit values

Reply:
it's not a big deal but you might as well put <80> before <128> in this list